### PR TITLE
feat(sbb-map-container): provide scroll up button

### PR DESCRIPTION
**Pull request by HendrikExtSBB**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

-

## Browsers

I tested the build on the following browsers:

- [x] Firefox Desktop
- [x] Chrome Desktop
- [x] Edge Desktop
- [x] Safari Desktop
- [x] Chrome Mobile
- [x] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/components/sbb-map-container/sbb-map-container.tsx [hunk 1]
```diff
@@ -1,20 +1,88 @@
...
         <div class="sbb-map-container__sidebar">
+          <span
+            ref={(el): void => {
+              this._intersector = el;
+            }}
+          ></span>
+
           <slot />
+
+          <div
```
[the remark this answers is not in the document]
Comment (HendrikExtSBB) [reply src/components/sbb-map-container/sbb-map-container.tsx]: I would say yes. Easyest way to place it inline centered above the content in my opinion.

File: src/components/sbb-map-container/readme.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
 # sbb-map-container
 
 This component is the layout container for the disruption map, the level 3 navigation and the
-future ATLAS. It provides two slots. One unnamed slot for the sidebar content and one named ("map") slot for map.
-On moblie the map is sticky above the sidebar and the sidebar content is scrolling over the map. On desktop is the sidebar
-and the map in a two column layout side by side.The component come along with a height calculation that subtracts the height
-of the header. The header height can be overriden with --sbb-map-container-margin-start if needed.
+future ATLAS. It provides three slots. One unnamed slot for the sidebar content, one named ("map") slot for map and one
```
[the remark this answers is not in the document]
Comment (HendrikExtSBB) [reply src/components/sbb-map-container/readme.md]: I removed the ATLAS reference and place an "e.g." for disruption map and lvl-3-nav. Fine like that for you?

**Comment by HendrikExtSBB** [comment]
> In order to fix the build you need to create an additional file like it was done for the MutationObserver (see `mutation-observer.ts`).

Moved the Intersection Observer to global/helpers. :)

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**